src/Components/PeoplePage/PeoplePage.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react";
+import MasonryGallery from "./MasonryComponent.js";
+import FilterDropdown from "./FilterDropdownComponent.js";
+import "./PeoplePage.scss"
+
+class PeoplePage extends Component {
+	constructor() {
+		super();
+		this.state = {
+		   people: [],
+		};
+	}
+
+	setFilter(filterBy){
+
+      const filteredPeople = filterBy === "all"
```

Comment:
Also the filteredPeople component we are passing the SetFilter Function so added the logic here to filterPeople and then update.

Note we are not actually filtering the data its just enabling and disabling the hover feature.